sympy/core/evalf.py

```diff
@@ -1391,6 +1391,28 @@ def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None,
             verbose=<bool>
                 Print debug information (default=False)
 
+        Note :
+        It is suggested to use expr.evalf(subs=...) over expr.subs(...).evalf()
+        or expr.evalf().subs(...) since expr.evalf(subs=...) avoids the loss of
+        significance caused due to naive substitution as the expression is run
+        through the evalf algorithm which takes into account, the various issues
+        that can cause loss of significance. Naive substitution can also include
+        substitution of only a few rather than all symbols in the expression.
+
+        Examples
+        ========
+
+        >>> import sympy
+        >>> x, y, z = sympy.symbols("x, y, z")
```

Comment:
The definition of symbols is space separated: `symbols('x y z')`.
You can also directly use `>>> from sympy.abc import x, y, z`.

Reply:
Done. @sidhantnagpal  Pls review the new PR. I have applied all the changes you suggested